stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -3707,7 +3774,7 @@ internal enum _Variant${Self}Storage<${TypeParametersDecl}> : _HashStorage {
       }
 
       let oldNativeStorage = asNative
-      let newNativeStorage = NativeStorage.create(minimumCapacity: minimumCapacity)
+      var newNativeStorage = NativeStorage.create(minimumCapacity: minimumCapacity)
```

Comment:
Going from a class to a struct made the compiler suddenly start caring about mutability.